pkg/webui/console/containers/api-keys-table/index.js

```diff
@@ -73,6 +73,17 @@ const headers = [
 ]
 
 export default class ApiKeysTable extends Component {
+  static propTypes = {
+    baseDataSelector: PropTypes.func.isRequired,
+    entityId: PropTypes.string,
```

Comment:
```suggestion
    entityId: PropTypes.string.isRequired,
```

Reply:
results in failed PropType, `entityId` is undefined

Reply:
As far as I can see the `id` prop is not even used by `FetchTable`. So you can delete it alltogether.